pkg/watch/health.go

```diff
@@ -131,6 +132,10 @@ func updatePods(
 	for _, pod := range current {
 		inReality := false
 		for _, man := range reality {
+			if man.PodUniqueKey != nil {
+				// We don't health check uuid pods
```

Comment:
:eyes:

How hard would it be to separate the components of the decision "Should I health check this pod?" from "Is this pod referenced by a UUID"? At the very least, I'd like to see this conditional encapsulated so we can do something more nuanced in the future.